### ISSUE TITLE
Accounts Receivable, Billing, and Financial issues-Federal Contact update

### ISSUE THREAD

**Issue by bbell0229** [ONRR/onrr.gov-site#1178]
Hello,

On the Payments page, 
https://www.onrr.gov/reportpay/PDFDocs/FSFederalCOAssignm.pdf, please update with the attached file. 

Please remove the prior attachment and replace with the new employee listing.

[finserAssign.pdf](https://github.com/ONRR/onrr.gov-site/files/7565609/finserAssign.pdf)

Thank you,

Brittney

**Comment by Maroyafaied** [comment]
@cvojslavek let me know if you have time to take this one.

**Comment by Maroyafaied** [comment]
You'll need to update the contacts spreadsheet as well if there are changes.

**Comment by cvojslavek** [comment]
@Maroyafaied I would have time after the ONRR Remote & Telework Informational Session and a team meeting till 3:30; however, I did an accessibility check, and the pdf is showing up with issues, see below screenshot with the issues:
![image](https://user-images.githubusercontent.com/89403899/142490239-e6350bd3-579b-4def-871e-0d031f0548c5.png)

**Comment by Maroyafaied** [comment]
@bbell0229 Please see @cvojslavek comment above with the attached picture. This document will need to be corrects and made accessible before we are able to post on the website. Thanks

**Comment by bbell0229** [comment]
[finserAssign.pdf](https://github.com/ONRR/onrr.gov-site/files/7566391/finserAssign.pdf)
I modified the last upload for this one. It should be good now. If not I may need assistance. I don't seem to have the accessibility check on my adobe (since it is the standard).

**Comment by Maroyafaied** [comment]
@bbell0229  The accessibility check is still showing one error, I believe Jessica @sheetsj has updated this document before so you may be able to ask her for help.

**Comment by bbell0229** [comment]
Jessica Sheets no longer works at ONRR.

Brittney A. Bell
Staff Accountant, Financial Services
Revenue, Reporting, & Compliance Management
Office of Natural Resources Revenue
Bldg 85, Denver Federal Center
Denver, CO 80225
1-800-433-9801 x 3381
303.231.3381
***@***.******@***.***>

From: Maroyafaied ***@***.***>
Sent: Thursday, November 18, 2021 3:28 PM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Bell, Brittney A ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Accounts Receivable, Billing, and Financial issues-Federal Contact update (Issue #1178)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@bbell0229<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbbell0229&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997727449%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=5GgghkVTt90%2FLw4%2Fzazfre54wT9LY%2BepM5h7Se2x9Pg%3D&reserved=0> The accessibility check is still showing one error, I believe Jessica @Sheetsj<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSheetsj&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997737404%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=uvHdLxIwIBF3oXOQj1bIWWgRbVu9P%2F0Y2EB5posZynI%3D&reserved=0> has updated this document before so you may be able to ask her for help.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F1178%23issuecomment-973324445&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997747366%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=9fod53zAGEm3I4%2FChLgvo3SSwH6Bk3%2BXetI9eMAoYaY%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUVLX3K2VA3H4B3G2NTOR7LUMV4WVANCNFSM5IKO3SJQ&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997757323%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=5UGjkiPU5W1qYDpD1rFrjeQmbNm%2BfZHjiuKNahAn2GA%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997767283%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Sf%2B4PaWLQnOdA93EdWiyw5a1eWjPDdZmb8NspdIhB0A%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cbrittney.bell%40onrr.gov%7Cdb98391f20954d3677ba08d9aae2b7ff%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637728712997767283%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=0Pi7eicGUMpjLz6%2B5DovUPhFzinIZQqGIl2BPkcgV%2BE%3D&reserved=0>.

**Comment by mcharg** [comment]
@bbell0229 The finserAssign.pdf doc was replaced with this doc as part of the 508 compliance project.
https://www.onrr.gov/ReportPay/PDFDocs/FSFederalCOAssignm.pdf

**Comment by mcharg** [comment]
@cthomasONRR and I both looked at this and neither of us could figure it out.  We reached out to Regina, but she's out until Monday.  We will revisit next week.
